apps/cookbook/src/app/showcase/form-field-showcase/form-field-showcase.component.ts

```diff
@@ -8,6 +7,7 @@ import { ShowcaseProperty } from '~/app/shared/showcase-properties/showcase-prop
   styleUrls: ['./form-field-showcase.component.scss'],
 })
 export class FormFieldShowcaseComponent {
+  size: string = 'lg';
```

Comment:
Consider strong typing:
```suggestion
  size: InputSize = InputSize.large;
```